staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go

```diff
@@ -360,46 +358,3 @@ func (h *testPluginHandler) Execute(executablePath string, cmdArgs, env []string
 	h.withEnv = env
 	return nil
 }
-
-func TestKubectlCommandHeadersHooks(t *testing.T) {
```

Comment:
What is the reason of the removal of this test?

Reply:
It verifies if the headers are added or not depending on the env var. Since the envvar is no more, we always add headers, it doesn't add any value to test that.